### ISSUE TITLE
set Initial state for code input 

### ISSUE THREAD

**Issue by alyona-mordas** [edwardfhsiao/react-codes-input#14]
Hi 
I am trying to set an initial state to the code input to a value from query parameters, but the submit button doesn't work in this case, even though I am able to see it properly typed in the field, the value is not bounded to the field.
Is there any work around for this ? 

`const [password, setPassword] = useState("xxxxxx");`

`                <ReactCodesInput
                  initialFocus={true}
                  wrapperRef={$passwordWrapperRef}
                  id="password"
                  codeLength={6}
                  type="alphanumeric"
                  hide={false}
                  value={password}
                  onChange={(res) => {
                    setPassword(res);
                  }}
                  customStyleComponent={{ maxWidth: "300px", margin: "0 auto" }}
                />
`

or even like this directly it doesn't work `value={"123456"}`

so basically unless I actually type it in, there is no way to set the value via code , is there ?

**Comment by edwardfxiao** [comment]
Good catch, try ```2.1.7```.

I have not tested for all specs. Some cases or scenarios may not be covered.

**Comment by edwardfxiao** [comment]
`2.1.8` handled hard setting input value only once, avoid some other issues

**Comment by alyona-mordas** [comment]
yes, it worked with 2.1.7 as well. Thank you
